packages/zosfiles/CHANGELOG.md

```diff
@@ -2,8 +2,10 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
-## `7.6.0`
+## Recent Changes
+- Enhancement: Modified the progress bar for USS file uploads to increase readibility. [#451](https://github.com/zowe/zowe-cli/issues/451)
```

Comment:
Q: should this be a BugFix / patch since it's not introducing new behavior ? 
Is there a new line missing between the header and the list ? (not like it really matters though, sorry for being picky 😢)